docs/conf.py

```diff
@@ -152,9 +152,9 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-    (master_doc, 'RITlugTeleirc.tex', 'RITlug Teleirc Documentation',
+    (master_doc, 'RITlugTeleIRC.tex', 'RITlug TeleIRC Documentation',
```

Comment:
Seeing what appear to be filenames change with a PR like this makes me mildly nervous. I just want to double check that we intended and accounted for it :)

Reply:
@thenaterhood what makes you nervous?

Reply:
This appears to be setting the target file for the LaTeX to be written to. Since this is the output file name, I don't think it is an issue. Thanks for pointing it out either way.